templates/zerver/reset_confirm.html

```diff
@@ -28,10 +28,12 @@ <h1>{{ _('Set a new password.') }}</h1>
                 </div>
 
                 <div class="input-box password-div">
-                    <label for="id_new_password1" class="">{{ _('Password') }}</label>
+                    <div class="password-container">
+                        <span><label for="id_new_password1" class="">{{ _('Password') }}</label></span>
+                        <span id="reset-password-limit-indicator" class="tippy-zulip-tooltip" data-tippy-content="{{ ('Maximum password length: 100 characters') }}" data-tippy-placement="top"></span>
```

Comment:
I am not sure what language we used for translation in portico pages, but you should atleast tag this for translation using `trans` tag like we do for other `data-tippy-content` attributes in this file.